README.md

```diff
@@ -0,0 +1,26 @@
+### Docker ###
+
+    Open your favorite Terminal and run these commands.
```

Comment:
Please remove indentation, because it makes formatting incorrect. You may check MarkDown formatting using https://dillinger.io/